apps/api/src/app/integrations/e2e/get-integration.e2e.ts

```diff
@@ -10,24 +10,35 @@ describe('Get Integrations - /integrations (GET)', function () {
     await session.initialize();
   });
 
-  it('should get newly created integration', async function () {
+  it('should retrieve all the integrations of all environments from an organization from the prefilled test data', async () => {
```

Comment:
I am replacing this existing test that just checked one integration from the prefilled test data for this one that does a deeper check and serves to prove the functionality of the endpoint even if we test again the test data.